runtime/renderer/xwalk_content_renderer_client.cc

```diff
@@ -156,6 +158,11 @@ void XWalkContentRendererClient::RenderViewCreated(
 #elif defined(OS_TIZEN)
   XWalkRenderViewExtTizen::RenderViewCreated(render_view);
 #endif
+  // TODO(sgurun) do not create a password autofill agent (change
```

Comment:
saurun is not you, correct?

I'd suggest to add comment to say the code is copied from `src/android_webview/renderer/aw_content_renderer_client.cc`, as long as the TODO.